receiver/cloudfoundryreceiver/config_test.go

```diff
@@ -96,6 +121,14 @@ func TestInvalidConfigValidation(t *testing.T) {
 	configuration.UAA.Password = ""
 	require.Error(t, configuration.Validate())
 
+	configuration = loadSuccessfulConfig(t)
+	configuration.RLPGateway.ShardID = ""
+	require.Error(t, configuration.Validate())
+
```

Comment:
```suggestion
```
It looks like we have two identical test cases here, should we delete one of the duplicates? Maybe there's a reason they were both here?